### DISCUSSION TITLE
Eligibility of Apprentices (German Azubi)

### DISCUSSION THREAD

**Discussion by Bloodiko** [community/community#53800] [GitHub Education]
Hello, 

i wanted to ask if really only academic students are eligible. 
Many places grant the benefits to job apprentices as well. 

I am currently still an apprentice for "Fachinformatik Systemintegration"  a german title, non academic but still the same category, basically. 
Last year i got approved to the Education Benefits, now i was asked to reapply, i am still apprentice, so i wanted to reapply. 
But none of the "Proofs" i upload seem to work. 

Issue is, i am only linked to the "general tips on how to approve", theres no error or something, its just denied. 

Approval: 
![image](https://user-images.githubusercontent.com/21989414/234623878-ba8fb2c7-d98e-4567-87c8-29d68319b870.png)

Denial: 
![image](https://user-images.githubusercontent.com/21989414/234624018-cbd4dfc9-2bff-4db0-87ce-dcd8ab8ebf51.png)

I am basically submitting the same document, with a newer date now, to proof that i still attend school. 
Thats why i am asking. 
* Did the Rules got more strict in the recent time ? 
* Am I no Longer eligible for education
* does the automatic check fail, because i do not own a facility email adress
* does the automatic check fail, because the document is in german, and not in english ? - hard to get english documents here in germany.....
* does the automatic check fail, because the "non-facility-email" has already been used before (it ran out, but its the same email, on the same account. i couldnt get the proof in time) 

I'd like to continue using the benefits until i graduate. 

Thanks, 

Bloodiko

**Comment by samus-aran** [comment]
Hi @Bloodiko 

Let's see how we can assist ⭐ 

> Did the Rules got more strict in the recent time ?

We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

> Am I no Longer eligible for education?

To be eligible for GitHub Global Campus, including GitHub Student Developer Pack and other benefits, you must:

- **Be currently enrolled in a degree or diploma granting course of study** such as a high school, secondary school, college, university, homeschool, or similar educational institution
- Have a verifiable school-issued email address or upload documents that prove your current student status
- Have a [GitHub personal account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
- Be at least 13 years old

So if you are still enrolled and meet the other (academic email is at sometimes not necessary but preferable) criteria then yes 👍 

> does the automatic check fail, because i do not own a facility email adress

This depends on the selected school.

> does the automatic check fail, because the document is in german, and not in english ? - hard to get english documents here in germany.....

The application process may require your evidence to be pre-translated. Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

> does the automatic check fail, because the "non-facility-email" has already been used before (it ran out, but its the same email, on the same account. i couldnt get the proof in time)

Not sure what you mean by this. Can you clarify?

**Reply by Bloodiko** [reply]
Thanks for the extensive response. 

>Not sure what you mean by this. Can you clarify?

As you can see in the two pictures, i had student pack until a few days ago. - i got an email to renew, but i was unable to re-apply in time, thus it ran out. ( I used my Gmail adress when i had it first ) - now when I select the same email adress, i get the Info that this e-mail already got a student discount. 

See Below, its the same Username though, same account, same everything. - Is this supposed to happen ? 
![image](https://user-images.githubusercontent.com/21989414/234716577-bf0a99e6-94d7-4795-850d-7cd9252ef65e.png)

But, i remember beeing on school ground the last time, when I got it approved. Maybe thats the reason, because I am submitting it from home, and the location differs. 
I will try again when i am next time in school. 

Also, adding a translation to the photo is not really applicable, since it probably reduces the quality of the photo overall, so that necessary infos may no longer be readable. 
I have no issues submitting multiple Photos with various documents, showing that I am student of a school, only using one seems hard. - I do have a student ID, but the Infos are in the front, while the date info is on the back. cant photo both at the same time. 

It may be necessary to add at least an optional second photo to the process.
